systems/node/node-gateway/pkg/rest/router.go

```diff
@@ -195,7 +212,7 @@ func (r *Router) postSystemPerformanceInfoHandler(c *gin.Context, req *StoreRunn
 		genCapps = append(genCapps, genCapp)
 	}
 
-	return r.clients.Health.StoreRunningAppsInfo(&healthPb.StoreRunningAppsInfoRequest{
+	return r.clients.health.StoreRunningAppsInfo(&healthPb.StoreRunningAppsInfoRequest{
```

Comment:
This should be :
`r.clients.Health.StoreRunningAppsInfo(`

Reply:
addressed